src/service/AppealStorageService.ts

```diff
@@ -0,0 +1,34 @@
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+
+import { Appeal } from 'app/models/Appeal';
+
+export class AppealStorageService {
+
+    constructor(private readonly uri: string) {
+        this.uri = uri;
+    }
+
+    public async save(appealData: Appeal, token: string): Promise<any> {
+
+        if (token == null) {
```

Comment:
It would make sense to add null check for appeal arg as well.

And you could rename arg to be just `appeal`.